src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="calendar calendar_start-day--fri calendar_month-length--28">
```

Comment:
Follow BEM naming convention
`block--modifier-name--modifier-value`

```suggestion
    <section class="calendar calendar--start-day--fri calendar--month-length--28">
```